python/ray/util/spark/cluster_init.py

```diff
@@ -921,7 +937,14 @@ def setup_ray_cluster(
             Default value is 1.0, minimum value is 0
 
     Returns:
-        The address of the initiated Ray cluster on spark.
+        A tuple of (address, remote_connection_address)
```

Comment:
You need to change the type annotation of the function as well.

Reply:
It's OK. We haven't announced GA for Ray on spark in our product side. :)